src/main.tsx

```diff
@@ -3,8 +3,12 @@ import { createRoot } from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
```

Comment:
On `main` branch when I enter `localhost:3000`, it redirects me to `localhost:3000/tracks`

On this branch it redirects me to `http://localhost:3000/?page=1&orderBy=asc&sortBy=artist` which is invalid page and I need to reload once more for app to redirect me correctly.

Also consider making default values unset to URL or maybe you've already considered it and decided to remain it in link, both options have its cons, pros: clean URL vs consistency if default URL params change